test/e2e/network/networking.go

```diff
@@ -226,6 +226,36 @@ var _ = SIGDescribe("Networking", func() {
 			config.DialFromEndpointContainer("udp", config.NodeIP, config.NodeUDPPort, config.MaxTries, 0, config.EndpointHostnames())
 		})
 
+		ginkgo.It("should function for multiple endpoint-Services with same selector", func() {
```

Comment:
I think it could be important here to note the original motivation for this test - ensuring that connectivity is retained after 1/2 Services targeting a Pod is deleted.

Reply:
That is a good point! I will add a clarifying comment.